File: Proyectos python (4)/adivinar numero.py
import random
cont=0
num=0
r=0
print("Programa que va a generar un numero aleatorio, y el usuario lo va a adivinar\n")
print("El numero esta entre 0 y 100\n")

x= random.randint(0 , 100)

while r == 0:

    num=(int(input("Ingrese un numero: \n")))
    cont=cont+1
    if (num>x):
        print("Menos\n")
    if (num<x):
        print("Mas\n")
    if(num==x):
        break
print("¡Haz adivinado!\n")
print("Numero de intentos",cont)

File: Proyectos C++ (4)/adivinar numero.cpp
#include <iostream>
#include <time.h>
#include <stdlib.h>

using namespace std;

int main ()
{
	int x, num, cont = 0;
	srand ((unsigned)time(NULL));
	x = rand()%(101);
	
	cout<<"Programa que va a generar un numero aleatorio y el usuario va a adivinarlo"<<endl;
	cout<<"El numero esta entre 0 y 100"<<endl;
	int a = 0;
	
	while (a == 0)
	{
		cout<<"Ingrese un numero: "<<endl;
		cin>>num;
		cont++;
		if(num>x)
			cout<<"Menos"<<endl;
		else if (num<x)
			cout<<"Mas"<<endl;
		else
		{
			cout<<"¡Haz acertado!"<<endl;
			cout<<"Numero de intentos: "<<cont<<endl;
			
			a = 1;
		}
	}
	
}
